projects/storefrontlib/src/lib/ui/pages/404-page/404-page.component.html

```diff
@@ -0,0 +1 @@
+<cx-page-layout></cx-page-layout>
```

Comment:
we want to remove the whole 404 page module. Use pageLayout to render it

Reply:
will fix! I thought we would want to add the component back